### ISSUE TITLE
Support react-native-sketch-canvas

### ISSUE THREAD

**Issue by syedibrahimt** [microsoft/react-native-windows#5975]
# Support `react-native-sketch-canvas` on RNW

## Basic Info
- Module Repo URL: [link to repo](https://github.com/terrylinla/react-native-sketch-canvas)
- Target version: react native - 0.63
- [ ] Issue exists on module repo (if so, [link]())

- [ ] Module has a Legacy/C# implementation (if so, [link]())

## Which app/customer is requesting this module?

BackPack - Knomadix corporation

## In what scenarios will this module be used?

- The end user can be able to write/ draw based on touch events or mouse events.
Example scenario: Student can be able to solve math problems by using the drawable canvas.

## Essential Features

1. The user can able to select the width of the stroke.
2. The user can be able to change the color of the ink stroke.
3. The developer needs the ability to add stroke/ delete stroke dynamically. 
4. Support iOS and Android.
5. Support drawing on an image
6. Use vector concept. So sketches won't be cropped in different sizes of canvas.
7. Can serialize path data to JSON. So it can sync other devices or someone else and continue to edit.

**Comment by creambyemute** [comment]
@chrisglein The main repo (terrylinla) is unfortunately not really maintained anymore.

I have a repo that is 2 commits ahead and fixes 2 **serious** problems on iOS (memory leaks and a fix for onPathChange not being fired on iOS): https://github.com/creambyemute/react-native-sketch-canvas

If this gets implemented for windows, would you either mind doing it on top of my fork (I know, unlikely), grab my 2 commits for iOS, or would there be a chance I'm allowed to grab the windows implementation into my repo? 

We're currently re-setting up our RN-Windows platform since VNext came out 💯, there are not many missing parts anymore, this is one of them.

Keep up the fantastic work on RN & RN-Windows 👍, love it!

**Comment by stmoy** [comment]
👋 - hi @creambyemute! Great to hear that you're working on supporting RNW!

We were worried that the main repo wasn't really maintained, but chose that one specifically because the customer requested that repo. 

@syedibrahimt - if you can move to the fork, we could port these changes over there. 

Either way, @creambyemute, please do feel free to grab the windows implementation and let us know if you have any troubles!

https://github.com/terrylinla/react-native-sketch-canvas/pull/182

**Comment by creambyemute** [comment]
@stmoy No worries, I already merged it in my repo and autolinking seemed to work 🔥 !
I haven't really maintained my repo either apart from the iOS Memory leak fixes and at the moment the original implementation + my iOS fixes work fine for me.

I'm a bit reluctant atm to change stuff there if not absolutely needed. Though I'll give the PR's in the original repo a look to see if there's something safe & worthwhile to merge.

I haven't come around to really test the implementation as I had to sort out RN-Gesture-Handler + React-Navigation problems first, but that's done now.
Should reach the Dashboard tomorrow after that I should be able to try out the RN-SketchCanvas implementation and will give feedback if I find anything 👍 .

 🔥 I Hope a lot of you RN-Windows ppl see this 🔥 
Honestly, it is fun (and daunting :-)) setting RN-Windows up on an already existing project and I just want to give Kudos to you how you are supporting/approaching community modules.
When `RN-Windows VNext` was announced I feared RN-Windows will slowly fade away, but it's better than ever before nowadays imo ❤️
